src/picknik_ur_gazebo_config/objectives/object_segmentation_3d.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<root BTCPP_format="4" main_tree_to_execute="Object Segmentation 3D">
+  <!-- ////////// -->
+  <BehaviorTree ID="Object Segmentation 3D" _description="Detect objects using a 2D image segmentation model and convert them to 3D masks and graspable objects." _favorite="false" _hardcoded="false">
+    <Control ID="Sequence" name="TopLevelSequence">
+      <Action ID="ClearSnapshot"/>
+      <Action ID="MoveToWaypoint" waypoint_name="Extended Right" planning_group_name="manipulator" controller_names="/joint_trajectory_controller /robotiq_gripper_controller" use_all_planners="false" constraints=""/>
+      <Action ID="GetCameraInfo" topic_name="/wrist_mounted_camera/color/camera_info" message_out="{camera_info}"/>
+      <Action ID="GetPointCloud" topic_name="/wrist_mounted_camera/depth/color/points" message_out="{point_cloud}"/>
+      <Action ID="GetImage" topic_name="/wrist_mounted_camera/color/image_raw" message_out="{image}"/>
+      <Action ID="GetMasks2DAction" image="{image}" action_name="/get_masks_2d_maskrcnn" min_confidence="0.8" max_nms_iou="0.8" min_relative_area="0" max_relative_area="0.8" timeout_sec="-1" masks2d="{masks2d}" valid_classes=""/>
```

Comment:
Yeah, I'm leaning towards "unnecessary complexity" only because I want to keep the `GetCameraInfo`, `GetPointCloud`, and `GetImage` Behaviors grouped in the 3D objective.

Reply:
That's understandable.  Maybe in the future we could make this into a subtree, `GetDepthImage` so it's more apparent when we need to call GetPointCloud and GetCameraInfo (when using 3D pipeline). Not sure how this would relate to `GetSynchronizedCameraTopics` or if that already covers our need. Anyways not important for this PR.

Reply:
`GetSynchronizedCameraTopics` is a viable alternative to these behaviors! I think we end up not using them as often because they can time out on slower setups, in CI, etc.